redbox/local.py

```diff
@@ -294,3 +329,79 @@ def rag_chat(self, chat_request: ChatRequest, callbacks: Optional[list[Callable]
                 SourceDocument.from_langchain_document(document=document) for document in response["input_documents"]
             ],
         )
+
+    def stuff_doc_summary(
+        self, summary: PromptTemplate, file_uuids: list[UUID], callbacks: Optional[list[Callable]] = []
+    ) -> ChatResponse:
+        documents: list[Document] = []
+        for file_uuid in file_uuids:
+            chunks = self.get_file_chunks(file_uuid=file_uuid)
+            document = Document(
+                page_content=" ".join([chunk.text for chunk in chunks]),
+                metadata=reduce(Metadata.merge, [chunk.metadata for chunk in chunks]),
```

Comment:
```suggestion
                metadata=reduce(Metadata.merge, (chunk.metadata for chunk in chunks)),
```

*really* nitpicking now!